### ISSUE TITLE
Add -q to kill-server

### ISSUE THREAD

**Issue by keelung-yang** [tmux/tmux#4281]
Hi,
I call `tmux kill-server` at the begining of my shell script and get a warning of
```bash
$ tmux kill-server
no server running on /tmp/tmux-1000/default
```
It's noisy. 
So, it's better if there is a `-q` argument to make kill-server quiet.

I know I can redirect it to /dev/null, but `-q` is a better way.

Thanks!

**Comment by nicm** [comment]
I don't think this is worth it when, as you say, you can just redirect to /dev/null.

**Comment by keelung-yang** [comment]
> I don't think this is worth it when, as you say, you can just redirect to /dev/null.

Then who need this `no server` warning ? 
```
If someone need, then it should be shown.
If soneone no need, then it shouldn't be shown.
If no one need, then it shouldn't be shown anymore.
```
So, before determine who need this warning, it's better to provide an option to users.
Redirecting isn't a elegant code in most of use cases.

Such as kill/pkill command, if no process, their is no warning.

**Comment by nicm** [comment]
The rule for Unix error reporting is no output on success, only on failure.

I don't know where you found your kill command:
~~~~
$ kill 1234
ksh: kill: 1234: No such process
$ kill -q 1234
ksh: kill: -q: unknown option
$

**Comment by kenlau46** [comment]
@keelung-yang , from the kill/pkill man page (Ubuntu 24.04)
```
       -q, --queue value
              Use sigqueue(3) rather than kill(2) and the value argument is used to specify an integer to be sent with the signal. If the receiving process has installed a handler for this signal using  the
              SA_SIGINFO flag to sigaction(2), then it can obtain this data via the si_value field of the siginfo_t structure.
```

In addition, if you want to define a `-q` option, you would need to defined what message should be quite and what should be not, instead of targeting a single output message

**Comment by keelung-yang** [comment]
> @keelung-yang , from the kill/pkill man page (Ubuntu 24.04)
> 
> ```
>        -q, --queue value
>               Use sigqueue(3) rather than kill(2) and the value argument is used to specify an integer to be sent with the signal. If the receiving process has installed a handler for this signal using  the
>               SA_SIGINFO flag to sigaction(2), then it can obtain this data via the si_value field of the siginfo_t structure.
> ```
> 
> In addition, if you want to define a `-q` option, you would need to defined what message should be quite and what should be not, instead of targeting a single output message

Then more complicated things should be considered as logging level in python.

The reason I fire this issue is, I get a shell script from NVIDIA in which too many redirections used for muting tmux error.
Why tmux users need more and more redirections ?
I don't want to add more redirections to that script which I've to update.

**Comment by nicm** [comment]
Unfortunately stderr handling is a big part of shell scripting whether you like it or not, but you could always do something like:
~~~~
tmux_ignore_error() {
    tmux "$@" 2>/dev/null || true
}
